apps/fz_http/lib/fz_http_web/live/setting_live/customization_live.ex

```diff
@@ -0,0 +1,78 @@
+defmodule FzHttpWeb.SettingLive.Customization do
+  @moduledoc """
+  Manages the app customizations.
+  """
+  use FzHttpWeb, :live_view
+
+  alias FzHttp.Conf
+
+  @max_logo_size 256 * 1024
```

Comment:
Wondering if 256 KB might be too small. Maybe bump to 1 MB?

Reply:
Naw, I really hope not. I just know that we'll get tons of users complaining that they can't upload their high DPI amazing company logo. Hah.

Hmm I was afraid of that... the DB solution is so easy but hard to cache. While uploading to the filesystem is hard (and creates scalability problems) but can be cached by nginx handily.

Reply:
The

Reply:
> The middle ground is of course caching in memory with persistent terms

Sounds good. This works for now 👍